prereise/gather/solardata/helpers.py

```diff
@@ -29,7 +29,7 @@ def to_reise(data):
         if i == 1:
             profile = data_tmp.T
         else:
-            profile = profile.append(data_tmp.T, sort=False, ignore_index=True)
+            profile = pd.concat([profile, data_tmp.T], sort=False, ignore_index=True)
```

Comment:
Just curious, what does warning it produce here? It seems to be equivalent.

Reply:
`append` seems to deprecated in favor of `concat`.

Reply:
I think @lanesmith encountered the same issue.